src/ripple/rpc/impl/TransactionSign.h

```diff
@@ -29,103 +30,38 @@ namespace RPC {
 
 namespace detail {
 
-// A class that allows these methods to be called with or without a
-// real NetworkOPs instance.  This allows for unit testing.
-class TxnSignApiFacade
-{
-private:
-    NetworkOPs* const netOPs_;
-    std::shared_ptr<ReadView const> ledger_;
-    AccountID accountID_;
-    std::shared_ptr<SLE const> sle_;
-
-public:
-    // Enum used to construct a Facade for unit tests.
-    enum NoNetworkOPs{
-        noNetOPs
-    };
-
-    TxnSignApiFacade () = delete;
-    TxnSignApiFacade (TxnSignApiFacade const&) = delete;
-    TxnSignApiFacade& operator= (TxnSignApiFacade const&) = delete;
-
-    // For use in non unit testing circumstances.
-    explicit TxnSignApiFacade (NetworkOPs& netOPs)
-    : netOPs_ (&netOPs)
-    { }
-
-    // For testTransactionRPC unit tests.
-    explicit TxnSignApiFacade (NoNetworkOPs noOPs)
-    : netOPs_ (nullptr) { }
-
-    // For testAutoFillFees unit tests.
-    TxnSignApiFacade (NoNetworkOPs noOPs, std::shared_ptr<ReadView const> ledger)
-    : netOPs_ (nullptr)
-    , ledger_ (ledger)
-    { }
-
-    void snapshotAccountState (AccountID const& accountID);
-
-    bool isValidAccount () const;
-
-    std::uint32_t getSeq () const;
-
-    boost::optional<STPathSet>
-    findPathsForOneIssuer (
-        AccountID const& dstAccountID,
-        Issue const& srcIssue,
-        STAmount const& dstAmount,
-        int searchLevel,
-        unsigned int const maxPaths,
-        STPathSet const& paths,
-        STPath& fullLiquidityPath) const;
-
-    void processTransaction (
-        Transaction::pointer& transaction,
-        bool bAdmin,
-        bool bLocal,
-        NetworkOPs::FailHard failType);
-
-    std::uint64_t scaleFeeBase (std::uint64_t fee) const;
-
-    std::uint64_t scaleFeeLoad (std::uint64_t fee, bool bAdmin) const;
-
-    bool hasAccountRoot () const;
-
-    error_code_i singleAcctMatchesPubKey (
-        RippleAddress const& publicKey) const;
-
-    error_code_i multiAcctMatchesPubKey (
-        AccountID const& acctID,
-        RippleAddress const& publicKey) const;
-
-    int getValidatedLedgerAge () const;
-
-    bool isLoadedCluster () const;
-};
-
 // A function to auto-fill fees.
-enum class AutoFill : unsigned char
-{
-    dont,
-    might
-};
-
 Json::Value checkFee (
```

Comment:
The `checkFee` function is only exposed in the header so it can be called by a unit test: https://github.com/scottschurr/rippled/blob/develop/src/ripple/rpc/tests/JSONRPC.test.cpp#L881.  So it's not really part of the public API.